docs/wxm-devices/D1-bundle/D1-troubleshooting.mdx

```diff
@@ -0,0 +1,125 @@
+
+Even weather stations, which are close to each other, may obtain slightly different measurements,
+especially for the variables of wind speed and wind direction. However, you should not expect significant deviations
+e.g., from a weather station, which is located within a distance of 1km/0.6mi.
+If there are deviations, please check [deployments examples for Proper Station Installation](/deployment-examples) for guidelines about the installation of the weather station.
+
+### One or more variables are extremely high, low, or constant for long periods of time.
+
+
+Change the batteries of the weather station. Make sure you use lithium non-rechargeable batteries, especially in case of cold weather environments.
+
+### Pressure on D1 Gateway appears to be very low relatively to other reference stations of national or private networks etc. 
+
+For now we present only the absolute pressure and not the Mean Sea Level Pressure (MSLP). That means that the displayed pressure are affected by the elevation of your area. We work to consider the elevation given by your GPS in order to show the MSLP, which is commonly used.
+
+
+## WiFi  Connectivity
+
+### No WiFi  connection
+
+Check for the WiFi  symbol on the top-right corner of the display.
+
+- If wireless connectivity is operational, the WiFi  icon will appear in green.
+- If the symbol appears in red, but you do remember configuring it successfully before, check that the D1 Gateway and Internet router are close enough.
+
+If you have never been able to configure WiFi  to a working state, make sure your WiFi  supports 2.4 GHz signals. The gateway does not support WiFi that uses the 5 GHz spectrum.
+
+Make sure you configured the correct SSID and password. Repeat the procedure if necessary to verify.
+
+ <hr />
+
+### Wifi Repeater
+Usually, using a WiFi repeater is not an issue, but in some cases it may affect your Gateway's connection. If you are experiencing issues with your D1 Gateway, try to connect it directly to your router and see if the problem persists. If it does not, 
+then the issue is with the repeater. If it does, please contact us through [support](https://support.weatherxm.com).
+
+ <hr />
+
+### D1 Gateway not working in 5 GHz WiFi.
+D1 Gateway is designed to work **only** at the 2.4GHz frequency. Please make sure the 2.4Ghz band on your router is enabled. If you are still experiencing issues, please contact us through [support](https://support.weatherxm.com).
+
+<hr />
+
+### I don't see the Wi-Fi name "WXM_WG1200" for initial setup on the app.
+
+It might take some time and attempts for it to connect. Please make sure that your Wi-Fi supports 2.4 GHz signals. You may also need to try from a different phone. If you are still experiencing issues, please contact us through [support](https://support.weatherxm.com).
+
+<hr />
+
+
+### Location not verified
```

Comment:
This should not be under "WiFi" issues. Create a different section as in M5 troubleshooting called "GPS Location" and add it there.